packages/api/src/graphql/database.ts

```diff
@@ -47,12 +47,45 @@ const DATABASE_STATUS = gql`
 `;
 
 const DATABASE_LIST = gql`
-  query GetDbList($search: SearchInput, $pagination: PaginationInput) {
-    dbList(search: $search, pagination: $pagination) {
+  query GetDbList($query: JSON, $pagination: PaginationInput) {
+    dbList(query: $query, pagination: $pagination) {
       databaseName
       databaseSize
       merkleHeight
-      collections
+      collections {
```

Comment:
so to be honest, collectionList, listCollection or collections, the naming convention messed up, not consistency

Reply:
It depends on the context: if it starts with a noun, it refers to graphql, if it starts with a verb, it refers to ts. If we changed the collections field here, we would either need to perform a mapping or ensure the names are consistent between the ts and grahpql code